internal/meta/req.go

```diff
@@ -35,6 +37,9 @@ func FromTransportRequest(ctx context.Context, req *transport.Request) yarpc.Req
 // ToTransportRequest fills the given transport request with information from
 // the given ReqMeta. The Context associated with the ReqMeta is returned.
 func ToTransportRequest(reqMeta yarpc.CallReqMeta, req *transport.Request) context.Context {
+	if reqMeta == nil {
+		log.Panic("reqMeta must not be nil")
```

Comment:
This is going to happen to people during runtime, so I'm not sure we should panic. 

@abhinav think we should just return an error instead?